cypress/integration/hitch-login_spec.js

```diff
@@ -592,14 +556,50 @@ describe('User time and day input, and about me section', () => {
 
     cy.get('[type="checkbox"]').eq(0).check({ force: true }).and('have.value', 'sunday');
 
-    cy.get('button').click();
+    cy.get('button').eq(1).click();
 
     cy.get('input').should('not.exist')
     cy.get('.bio__header').contains('About Me');
 
     cy.get('textarea').type('Hello world');
     cy.get('textarea').contains('Hello world');
 
-    cy.get('button').click();
+    cy.get('button').eq(1).contains('Submit')
+
+    cy.get('button').eq(1).click();
+  });
+});
+
+describe('Back button', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000/register');
+
+    cy.get('input').should('have.length', 3);
+    cy.get('.register-form__header').contains('Name');
   });
-});
\ No newline at end of file
+
+  it('Should not be present on the first page of form', () => {
+    cy.get("registration__back-btn").should('not.exist')
+  })
+
+  it('Should bring you back one page when clicked', () => {
+    cy.get('input').eq(0)
+      .type('Billy')
+      .should('have.value', 'Billy')
+    cy.get('input').eq(1)
+      .type('BillyBob33@gmail.com')
+      .should('have.value', 'BillyBob33@gmail.com')
+    cy.get('input').eq(2)
+      .type('billyB')
+      .should('have.value', 'billyB')
+
+    cy.get('button').eq(1).click();
+
+    cy.get(".registration__back-btn").contains('Back');
+
+    cy.get(".registration__back-btn").click();
+
+    cy.get('input').should('have.length', 3);
+    cy.get('.register-form__header').contains('Name');
+  })
+})
```

Comment:
Thanks for updating the tests, I think we pretty much have all happy/ sad paths covered!